lib-core/src/widgets/BaseWidget.tsx

```diff
@@ -26,13 +25,11 @@ export class BaseWidget<P extends BaseWidgetProps = BaseWidgetProps, S = any> ex
 	}
 
 	bem(selector: string): string {
-		return (this.props.baseClass || this.className) + selector + " ";
+		return (this.props.baseClass || this.className) + selector + ' ';
 	}
 
 	getClassName(): string {
-		return (
-			(this.props.baseClass || this.className) + " " + (this.props.className ? this.props.className + " " : "")
-		);
+		return (this.props.baseClass || this.className) + ' ' + (this.props.className ? this.props.className + ' ' : '');
```

Comment:
There is package for concatinating classNames: https://www.npmjs.com/package/classnames

Reply:
this is going to be fixed when we move to emotion